src/platform/packages/shared/kbn-cell-actions/.eslintrc.js

```diff
@@ -73,11 +73,30 @@ const path = require('path');
 const minimatch = require('minimatch');
 
 /** @type {Array.<RestrictedImportPath>} */
-const RESTRICTED_IMPORTS = [
+const RESTRICTED_IMPORTS_PATHS = [
```

Comment:
@kapral18 , is there a way to DRY this script to avoid repeating the almost same script in each `.eslintrc` does